File: problema9.hs
import Data.Char (toLower)

pizza :: Bool -> String -> (String, String)
pizza vegetariana ingrediente=
  let tipo=if vegetariana then "vegetariana" else "no vegetariana"
      siNo=if vegetariana then "si" else "no"
      descripcion="Pizza "++tipo++" con mozzarella, tomate y "++ingrediente
  in (descripcion, siNo)

main :: IO ()
main=do
  putStrLn "¿Pizza vegetariana? (si/no):"
  resp<-getLine
  let veg=map toLower resp=="si"

  putStrLn "Elige un ingrediente adicional:"
  ing<-getLine

  let (descripcion, esVegSiNo)=pizza veg ing
  putStrLn descripcion
  putStrLn ("¿Es vegetariana? "++esVegSiNo)

File: problema9.py
print("Bienvenido a la pizzería don José")

vegetariana=input("¿Quieres una pizza vegetariana? (si/no): ")

if vegetariana.lower()=="si":
    print("Ingredientes disponibles: ")
    print("1. pimiento")
    print("2. cebollin")
    opcion = int(input("Elige un ingrediente (1-2): "))
    if opcion==1:
        ingrediente="pimiento"
    else:
        ingrediente="cebollin"
    tipo="vegetariana"
else:
    print("Ingredientes disponibles: ")
    print("1. pepperoni")
    print("2. jamón")
    print("3. salami")
    opcion = int(input("Elige un ingrediente (1-3): "))
    if opcion==1:
        ingrediente="pepperoni"
    elif opcion==2:
        ingrediente="jamón"
    else:
        ingrediente="salami"
    tipo="no vegetariana"

print(f"\nTu pizza es {tipo}.")
print(f"Lleva mozzarella, tomate y {ingrediente}.")
